pkgs/development/tools/lc3tools/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, unzip, flex, tk, ncurses, readline }:
+
+stdenv.mkDerivation {
+  pname = "lc3tools";
+  version = "0.12";
+
+  src = fetchurl {
+    url = "https://highered.mheducation.com/sites/dl/free/0072467509/104652/lc3tools_v12.zip";
```

Comment:
Is there any URL which uses the version? Otherwise auto updates do not work.

Reply:
There hasn't been an update since 2008. I can potentially ask Lumetta for newer source (he's my professor).

Reply:
I am not sure what to do here but packaging software from 2008 does not feel right to me.

Reply:
If it still builds, why not?